modules/log-collection-rbac-permissions.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
```

Comment:
I defer here to  you if we wish to make mention this is only required for the multi CLF mode. Legacy mode will already allow creation of the legacy CLF without the permissions.

Reply:
Good call